index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+  <title></title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+
+<body>
+   <h1> Expenses<div id="expense-count"></div></h1>
+   <h3> Current Expenses :<span id="total"></span> </h3>
+   <br>
+
+
+  <input type="number" placeholder="amount" class="amt">
+  <input type="text" placeholder="description" class="desc">
+  <button id="expense">Add Expense</button>
+
+
+  <table align="center">
```

Comment:
Make sure you write your tables correctly, indent child elements and close your tags!